Added a warning for the AP CSP create performance task (#51298)
code-dot-org/code-dot-org: .github/pull_request_template.md

## Warning!!

> The AP CSP Create Performance Task is in progress from April 1 - May 1 2023. Please

Reviewer: ```suggestion
The [AP CSP Exams](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) are currently in progress from April 1 - May 1 2023. Please
```
